traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -51,8 +51,10 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
+import java.io.UnsupportedEncodingException;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.net.URLEncoder;
```

Comment:
unused import

Reply:
I'm actually pretty amazed that you notice stuff like this. Fixed ;)

Reply:
@kjurkovic use F2 in android studio in your changed files to jump to lint errors and try to send them without lint error :D although i myself forget to do it some times and Gennady finds my lint errors :D